### DISCUSSION TITLE
ID3 Tag Import AutoCue Marker and more

### DISCUSSION THREAD

**Discussion by Radiotom** [AzuraCast/AzuraCast#5918] [Ideas]
Would it be nice to get a funktion to import or export the CueSheet of the Music into the MP3 ID3 Tag?
I woult like to export my whole CueSheet Data from mAirList (CueIn, CueOut, Fade, etc) into AzuraCast.

In case of a complete Music transfer, i use CSV Import, but if i update some Titles, it would be nice that AC look into the ID3 Tags and check if theres CueSheets Data to import.

Thanks

PS: Sorry fo my bad english.

**Comment by Moonbase59** [comment]
That’d be something! Does AzuraCast actually use the same tags mAirList uses for that purpose? I’m not aware of any standard here, everyone seems to use their own creations…

I suspect mAirList is not very well-known outside Germany, so let me mention it is a great (Windows-based) radio automation, studio integration and broadcasting system used by many professional and semi-professional broadcasters around here. MAirList would be one of the only two reasons I’d ever use a Windows machine again for. (The other is InDesign.)

Doing some things (like tags) in a uniform way would be just great.

**Reply by Radiotom** [reply]
It is, mAirList is a powerful Radio Broadcast Software based on Windows. It is used by many professional Broadcast Stations and semi professional Broadcasters around the World.

The features about the ID3 Tag export is a special way to save all Data's (also Cuepoits) sideways around the main local or Network Database.

I tried to complie the Timestamps to the Azuracast Format over CSV Export but it seems not easy because AC uses a different Format. 

But it's just an idea.
